build/pipelines/release.yml

```
@@ -234,6 +234,10 @@ jobs:
         TargetFolder: $(Build.ArtifactStagingDirectory)/appx
         OverWrite: true
         flattenFolders: true
+    - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+      displayName: 'Manifest Generation Task'

```

```suggestion
      displayName: 'Generate SBOM manifest'
```

nit: technically, display names should be more verby. "Do the thing" or "Publish the frob" or "Frob the bar"

---

OK I will Frob the bar.